docs/dyndns/update-api.rst

```diff
@@ -172,12 +172,9 @@ affiliated with the respective account (see :ref:`manage-tokens` for details.)
 stand-in for an IPv6 address. Replace those (including the ``<`` and ``>``)
 with your respective values.
 
+**Examples with headers** (recommended)
```

Comment:
Not sure if I understood it correctly, but I think that the header version is more secure, since the token does not end up in your logs?